io/eolearn/io/sentinelhub_service.py

```diff
@@ -34,7 +37,9 @@ class SentinelHubOGCInput(EOTask):
     :type size_y: int or str, depends on the service_type
     :param maxcc: maximum accepted cloud coverage of an image. Float between 0.0 and 1.0. Default is ``1.0``.
     :type maxcc: float
-    :param image_format: format of the returned image by the Sentinel Hub's WMS getMap service. Default is 32-bit TIFF.
+    :param image_format: format of the returned image by the Sentinel Hub's WMS getMap service. Default is `None` which
```

Comment:
Why is `image_format` parameter set to `None` by default here, if actually it is set to `MimeType.TIFF_d16` later on? Why not explicitly set the default value here so that later on the checks (if ... None ... else ...)?

Reply:
Will have to check. It is quite some time since this PR has been made (and closed).